src/types.ts

```diff
@@ -645,12 +645,13 @@ export interface NearSdk {
 }
 
 export interface KycDaoContract extends Contract {
-  mint: (_: {
+  mint_with_code: (_: {
     args: Record<string, unknown>;
     gas: string;
     amount: string;
     callbackUrl?: string;
   }) => Promise<unknown>;
+  get_required_mint_cost_for_code: (_: { args: Record<string, unknown> }) => Promise<string>;
```

Comment:
`args` could be typed better as `{ auth_code: number, dst: string }` I guess in our case.